apps/workspace-host/src/interface.ts

```diff
@@ -33,6 +33,43 @@ import { config, loadConfig } from './lib/config.js';
 import { parseDockerLogs } from './lib/docker.js';
 import { REPOSITORY_ROOT_PATH, APP_ROOT_PATH } from './lib/paths.js';
 import * as socketServer from './lib/socket-server.js';
+let server: http.Server | undefined;
```

Comment:
Nit, maybe keep this after the types and include a newline to delineate for other types/imports.